src/docs/markdown/download.md

```diff
@@ -10,7 +10,7 @@ All our [official distributions](https://github.com/caddyserver/dist) come with
 
 ## Static binaries
 
-You can **[download Caddy from GitHub](https://github.com/caddyserver/caddy/releases)**, where new releases are immediately published.
+You can **[download Caddy from GitHub](https://github.com/caddyserver/caddy/releases)**, where new releases are immediately published, and place it in your PATH.
```

Comment:
This is probably important; but the below steps don't explain how to do this, and I'm not really sure that's the right thing for our docs to do anyway, since that's basically "how to use your computer" more than "how to use Caddy" -- we can keep this change, but I'm worried it will confuse people when they run the command below and it doesn't appear in their PATH.

Reply:
I remember a wise man once said "many of the most common issues come from people not knowing how to use their computer".

I figure that at least then they'll know that there's such thing as a PATH and that caddy should be in it and if they google "how to put caddy in path" the first hit (for me at least) goes to the install docs, here: https://caddyserver.com/docs/install

I don't feel that strongly about it, but I think it's a nice touch.